src/explorer/CosmosDBTreeItem.ts

```diff
@@ -6,26 +6,36 @@
 import * as vscode from 'vscode';
 import { SiteClient } from 'vscode-azureappservice';
 import { IAzureNode, IAzureParentTreeItem, IAzureTreeItem } from 'vscode-azureextensionui';
+import { IConnections } from '../commands/connections/IConnections';
+import * as constants from '../constants';
+import { ConnectionAccountDatabaseTreeItem } from './ConnectionAccountDatabaseTreeItem';
 
 export class CosmosDBTreeItem implements IAzureParentTreeItem {
-    public static contextValue: string = 'CosmosDBConnection';
+    public static contextValue: string = 'сosmosDBConnection';
     public readonly contextValue: string = CosmosDBTreeItem.contextValue;
     public readonly label: string = 'Cosmos DB';
-
     constructor(readonly client: SiteClient) {
     }
 
     public async loadMoreChildren(_node: IAzureNode<IAzureTreeItem>, _clearCache: boolean): Promise<IAzureTreeItem[]> {
         const cosmosDB = vscode.extensions.getExtension('ms-azuretools.vscode-cosmosdb');
         if (!cosmosDB) {
             return [{
-                contextValue: 'InstallcosmosDBExtension',
+                contextValue: 'InstallCosmosDBExtension',
                 label: 'Install Cosmos DB Extension...',
                 commandId: 'appService.InstallCosmosDBExtension',
                 isAncestorOf: () => { return false; }
             }];
         }
-        throw new Error('Method not implemented.');
+        const workspaceConfig = vscode.workspace.getConfiguration(constants.extensionPrefix);
```

Comment:
If there are no connections, you should return a custom node that says "Add Cosmos DB Connection..." similar to your custom node above for 'Install Cosmos DB Extension...'